app/HutchAgent/Services/CrateMergerService.cs

```diff
@@ -8,6 +8,7 @@ namespace HutchAgent.Services;
 /// </summary>
 public class CrateMergerService
 {
+  private readonly string _pathToOutputDir = Path.Combine("Data","outputs");
```

Comment:
Was the name of the data directory `Data` or `data`? Another branch I'm on has it as `data` and I don't know what the agreed one was

Reply:
I'm pretty sure it's lowercase, and also it's not an "agreed" upon one from an internal or project perspective, right? it's the BagIt spec.

https://en.wikipedia.org/wiki/BagIt#Specification

Reply:
Ahh cool. The confusion might have been my fault because a couple of tests have "Data" in them, and I wrote them without bagit in mind.